pkg/sessions.go

```diff
@@ -457,6 +478,22 @@ func createEC2Session(sess *session.Session, region *string, role Role, fips boo
 	return ec2.New(sess, setSTSCreds(sess, config, role))
 }
 
+func createPrometheusSession(sess *session.Session, region *string, role Role, fips bool, isDebugEnabled bool) prometheusserviceiface.PrometheusServiceAPI {
+	fmt.Println("Creating prometheus sessions")
```

Comment:
This is spamming yace logs and breaks structured json log parsing.

Reply:
Fixed in https://github.com/nerdswords/yet-another-cloudwatch-exporter/pull/715